browser/main/Detail/MarkdownNoteDetail.js

```diff
@@ -262,6 +263,32 @@ class MarkdownNoteDetail extends React.Component {
     ee.emit('print')
   }
 
+  renderEditor () {
+    const { config } = this.props
+    const editorType = config.editor.type
+    let editor
+    if (editorType === 'DEFAULT') {
+      editor =
+        <MarkdownEditor
+          ref='content'
+          styleName='body-noteEditor'
+          config={config}
+          value={this.state.note.content}
+          storageKey={this.state.note.storage}
+          onChange={(e) => this.handleChange(e)}
+          ignorePreviewPointerEvents={this.props.ignorePreviewPointerEvents}
+        />
+    } else if (editorType === 'WYSIWYG') {
+      editor =
+        <WYSIWYGEditor
+          ref='content'
+          value={this.state.note.content}
+          config={config}
+          onChange={(e) => this.handleChange(e)} />
+    }
+    return editor
```

Comment:
if `editorType` is invalid value, this function will return null
I think assign `MarkdownEditor` to `editor` as a default value is more safe

Reply:
8a472b4 !